examples/logs.json

```diff
@@ -27,8 +27,9 @@
           },
           "logRecords": [
             {
-              "timeUnixNano": 1544712660300000000,
```

Comment:
The numbers [are acceptable when decoding](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md#json-protobuf-encoding) 64 bit int fields (so not completely wrong as an example), but I agree it is better to show examples that match our encoding requirements too.